packages/cli/src/config/webpack.config.common.js

```diff
@@ -125,6 +126,60 @@ module.exports = ({ config, context }) => {
         loader: 'file-loader'
       }]
     },
+    optimization: {
+      splitChunks: {
+        chunks: 'all',
+        minSize: 30000,
+        maxSize: 0,
+        minChunks: 1,
+        maxAsyncRequests: 6,
+        maxInitialRequests: 4,
+        automaticNameDelimiter: '~',
+        automaticNameMaxLength: 30,
+        cacheGroups: {
+          about: {
+            test(module, chunks) {
+              const regexp = /about/i;
+              const result = `${module.name || ''}`.match(regexp) || chunks.some(chunk => `${chunk.name || ''}`.match(regexp));
+
```

Comment:
Yes it's hardcoded into 4 cache groups, but ideally we could generate `cacheGroups` based on the graph.  This is utilizing our newly generated chunk names `page--[mainroute]--[page-name].js` which we create when the graph is filled.

Reply:
Or better yet, why don't we just move this to the `greenwood.config.js` and allow people to set their own chunks based on our generated graph chunk names.